sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Recovery Services Backup Resources.")]
 
 [assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.3.0")]
+[assembly: AssemblyFileVersion("4.0.4.0")]
```

Comment:
Here too. I'll just update it for you
```suggestion
[assembly: AssemblyFileVersion("4.1.0.0")]
```